packages/atomic/src/components/common/result-list/grid-display-results.tsx

```diff
@@ -14,17 +13,12 @@ export const GridDisplayResults: FunctionalComponent<ResultListDisplayProps> = (
       <div
         part="result-list-grid-clickable-container outline"
         ref={(element) => props.setNewResultRef(element!, index)}
+        onClick={(event) => {
+          event.preventDefault();
+          interactiveResult.select();
+          window.open(unfoldedResult.clickUri, props.target, 'noopener');
+        }}
```

Comment:
I just realized that we had an accessibility issue here, which isn't affected/fixed by these changes.

# The problem
Neither the new structure nor the old structure conveys to accessibility tools that results are clickable. For the most part, this is fine, since the template should contain another link (usually the title), which has the same purpose and conveys that they are clickable.

However, due to the whole result in grids being clickable already, some implementers may find it redundant to use `atomic-result-link` as a title, and may choose to use `atomic-result-text` instead. Some (or even most/all) accessibility tools should still be able to open the result, but they won't tell the users that any interactions are available.

# The fix
I think a good fix for this would be to add a [skip link](https://webaim.org/techniques/skipnav/) at the beginning of results.

Essentially, this would involve keeping the old `LinkWithResultAnalytics` that you removed below, but completely changing its CSS so it doesn't cover the result and instead behaves like a skip link.

You could also change the behaviour of the `onClick` event here so that it activates the link instead.

Reply:
We should make sure it's possible to open the link in a new tab with ctrl+click and mouse3 even if the `target` is `_self`.